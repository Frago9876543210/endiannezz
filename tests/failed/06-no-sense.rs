use endiannezz::Io;

#[derive(Io)]
#[endian(little)]
struct Foo {
    #[endian(little)]
    a: u32,
}

#[derive(Io)]
#[endian(le)]
struct Bar {
    #[endian(little)]
    a: u32,
}

fn main() {}
