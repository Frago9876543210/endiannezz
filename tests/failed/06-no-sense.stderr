error: this attribute does not make sense
 --> $DIR/06-no-sense.rs:6:5
  |
6 |     #[endian(little)]
  |     ^^^^^^^^^^^^^^^^^

error: this attribute does not make sense
  --> $DIR/06-no-sense.rs:13:5
   |
13 |     #[endian(little)]
   |     ^^^^^^^^^^^^^^^^^
